_includes/default-header.html

```diff
@@ -45,7 +45,7 @@
           </div>
```

Comment:
It might be good to modify the navbar above.

OLS-8 should be maybe removed. We could also then have a menu for each of the 3 pillars

Reply:
Definitely! I have just added a new navbar dropdown that contains our 3 pillars.
Since this captures the consulting service as well (under _open incubator_), I think we can remove the *Services* navbar-link. What do you think?

Reply:
![Screenshot (592)](https://github.com/open-life-science/open-life-science.github.io/assets/105166953/b35b690b-c06c-4b39-8144-9eb8c70beef3)